R/ReportCard.R

```diff
@@ -60,6 +60,18 @@ ReportCard <- R6::R6Class( # nolint: object_name_linter.
       private$content <- append(private$content, TextBlock$new(text, style))
       invisible(self)
     },
+    #' @description Appends a `rmarkdown` R chunk to this `ReportCard`.
+    #'
+    #' @param text (`character(0)` or `character(1)`) the text
+    #' @param ... any `rmarkdown` R chunk parameter and its value.
+    #' @return invisibly self
+    #' @examples
+    #' card <- ReportCard$new()$append_rcode("2+2", echo = FALSE)
+    #'
+    append_rcode = function(text, ...) {
```

Comment:
I'm thinking `code` or `src` (like `TealReportCard`) would be more clear here.

Reply:
The src is a show r code NEST related shortcut where as here we enable to add any r code.
Why r code not a code as we clearly communicating only r code is supported. 
In r markdown the other languages like python could be easily used too, I think only python could be added in future and with low probability. Then better to have a detail named method here.